games-roguelike/zangband/zangband-2.7.4c.ebuild

```diff
@@ -1,9 +1,9 @@
-# Copyright 1999-2019 Gentoo Authors
+# Copyright 1999-2021 Gentoo Authors
 # Distributed under the terms of the GNU General Public License v2
 
-EAPI=5
+EAPI=7
 
-inherit autotools eutils games
+inherit autotools multilib
```

Comment:
`multilib` shouldn't be required here too.

Reply:
Ditto: always audit the list of eclasses for unused inherits and sort (unless it affects phases).